roles/foreman_provisioning_infrastructure/tasks/main.yml

```diff
@@ -1,126 +1,64 @@
 ---
 # Make sure admin does not default to any taxonomy
 - name: 'disable default context for admin'
+  # TODO: how to pass org and location IDs 0 to FAM?
```

Comment:
I couldn't find this on https://theforeman.github.io/foreman-ansible-modules/develop/plugins/user_module.html#ansible-collections-theforeman-foreman-user-module and I'm wonder how this even works.